config/environments/development.rb

```diff
@@ -47,6 +47,12 @@
   config.action_mailer.perform_deliveries = true
 
 
+  # allow Mattermost slash commands
+  config.mattermost_settings = {
```

Comment:
We could put this in config/application.rb to eliminate the duplication with production.rb.